src/app/ConcreteClusterPath.h

```diff
@@ -52,7 +52,7 @@ struct ConcreteClusterPath
     // to alignment requirements it's "free" in the sense of not needing more
     // memory to put it here.  But we don't initialize it, because that
```

Comment:
this note no longer matches reality. Does this actually increase code sise?

Reply:
I had not read the comment ... thank you. Yes, it did increase the size.
Reverted and made the comment even clearer now.